app/scripts/controllers/pod.js

```diff
@@ -191,6 +193,10 @@ angular.module('openshiftConsole')
       return terminals;
     };
 
+    var updateContainersYet = function(pod) {
+      $scope.noContainersYet = $scope.containersRunning(pod.status.containerStatuses) === 0; 
```

Comment:
If it would be helpful, ping me for an overview of our e2e test setup to automate this.  Might be helpful to ensure you catch all the cases.  Ive been on a "we should test more" kick and am willing to hash that out a bit. :smile: